src/Illuminate/Console/Concerns/PromptsForMissingInput.php

```diff
@@ -37,7 +37,7 @@ protected function interact(InputInterface $input, OutputInterface $output)
     protected function promptForMissingArguments(InputInterface $input, OutputInterface $output)
     {
         $prompted = collect($this->getDefinition()->getArguments())
-            ->filter(fn ($argument) => $argument->isRequired() && is_null($input->getArgument($argument->getName())))
+            ->filter(fn ($argument) => $argument->isRequired() && empty($input->getArgument($argument->getName())))
```

Comment:
I'm concerned that `empty` is too loose. For example, someone could pass a value of `0`, and this would evaluate true. I think it would need to be something like:

```suggestion
            ->filter(fn ($argument) => $argument->isRequired() && (is_null($input->getArgument($argument->getName())) || (is_array($input->getArgument($argument->getName())) && count($input->getArgument($argument->getName())) === 0)))
```